workflow/scripts/get-reads.py

```
@@ -0,0 +1,41 @@
+import sys
+
+sys.stderr = open(snakemake.log[0], "w")
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)